src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -27,6 +32,14 @@ import { Banner } from "../Layout/Banner";
 import { ImportantMessageContentContext } from "../../context/importantMessage";
 import { NotEligibleCard } from "./NotEligibleCard";
 import { KeyboardAvoidingScrollView } from "../Layout/KeyboardAvoidingScrollView";
+import { navigateHome, replaceRoute } from "../../common/navigation";
+import { NavigationParams, NavigationRoute } from "react-navigation";
+import { NavigationDrawerProp } from "react-navigation-drawer";
+
+type CustomerQuotaProps = {
+  navigation: NavigationDrawerProp<NavigationRoute, NavigationParams>;
+  navIds: Array<string>;
```

Comment:
Done changing to `string[]`
`ids` would have been a more suitable name but it conflicted with the states:
`const [ids, setIds] = useState<string[]>(navIds);`

Any suitable name you have in mind?

Reply:
Hmm was thinking we can pass in as `ids` then alias it on line 78, so the external contract is still `ids` rather than `navIds`
```
export const CustomerQuotaScreen: FunctionComponent<CustomerQuotaProps> = ({
  navigation,
  ids: navIds
})
```